### PR TITLE
CODE RUB: Order Crud Operations For Teacher

### PR DESCRIPTION
**Pull request by nedataghiparast**

### CODE REVIEW

**Review by ShriHumrudha — CHANGES_REQUESTED** [review]

File: OtripleS.Web.Api/Services/Foundations/Teachers/TeacherService.cs [hunk 1]
```diff
@@ -47,34 +50,31 @@ public partial class TeacherService : ITeacherService
...
         });
 
         public ValueTask<Teacher> ModifyTeacherAsync(Teacher teacher) =>
         TryCatch(async () =>
         {
-            ValidateTeacherOnModify(teacher);
-            Teacher maybeTeacher = await this.storageBroker.SelectTeacherByIdAsync(teacher.Id);
-            ValidateStorageTeacher(maybeTeacher, teacher.Id);
-            ValidateAgainstStorageTeacherOnModify(inputTeacher: teacher, storageTeacher: maybeTeacher);
+             ValidateTeacherOnModify(teacher);
```
Comment (ShriHumrudha) [inline OtripleS.Web.Api/Services/Foundations/Teachers/TeacherService.cs:59]: This needs correction
image

**Review by ShriHumrudha — CHANGES_REQUESTED** [review]

File: OtripleS.Web.Api/Services/Foundations/Teachers/TeacherService.cs (hunk 1 above)
Comment (nedataghiparast) [reply OtripleS.Web.Api/Services/Foundations/Teachers/TeacherService.cs:59]: @ShriHumrudha Conflict is resolved

**Review by ShriHumrudha — CHANGES_REQUESTED** [review]
Fix the build issue

**Comment by ShriHumrudha** [comment]
Closing due to inactivity.

**Pull request closed without merging**